### PR TITLE
Creating a page presenting the maintainers

### PR DESCRIPTION
**Pull request by EgbertRijke**

### CODE REVIEW

File: src/SUMMARY.lagda.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - [Agda-UniMath](README.md)
   - [Home](README.md)
```
Comment (fredrik-bakke) [inline src/SUMMARY.lagda.md:6]: I think these should link to `SUMMARY.md`. Could you fix this in the same PR? :)

Comment (EgbertRijke) [reply src/SUMMARY.lagda.md:6]: Are you sure? The summary page only contains the list of files, but the home page (README) displays a few more things before it displays SUMMARY.

Comment (fredrik-bakke) [reply src/SUMMARY.lagda.md:6]: At least these links don't work for me. Do they work for you? Maybe the issue is that the README file is not properly uploaded?

Comment (EgbertRijke) [reply src/SUMMARY.lagda.md:6]: They just bring me back to the home page, as they should. @jonaprieto when you have time can you take a quick look to see if there is any issue? It's not urgent.

Comment (fredrik-bakke) [reply src/SUMMARY.lagda.md:6]: Oh, I see. Then I apologize for bringing it up here.

Comment (EgbertRijke) [reply src/SUMMARY.lagda.md:6]: No need to apologize. Thanks for bringing it up!

**Comment by jonaprieto** [comment]
It looks good to me.

**Review by jonaprieto — APPROVED** [review]

**Pull request merged**